projects/components/src/download-file/download-file.component.ts

```diff
@@ -45,31 +42,25 @@ export class DownloadJsonComponent {
 
   public triggerDownload(): void {
     this.dataLoading = true;
-    this.dataSource
-      .pipe(
-        take(1),
-        catchError(() => this.notificationService.createFailureToast('Download failed')),
-        finalize(() => {
-          this.dataLoading = false;
-          this.changeDetector.detectChanges();
-        })
-      )
-      .subscribe((data: unknown) => {
-        if (typeof data === 'string') {
-          this.downloadData(data);
-        } else {
-          this.downloadData(JSON.stringify(data));
-        }
-      });
+    this.metadata!.dataSource.pipe(
+      take(1),
+      catchError(() => this.notificationService.createFailureToast('Download failed')),
+      finalize(() => {
+        this.dataLoading = false;
+        this.changeDetector.detectChanges();
+      })
+    ).subscribe((data: string) => {
+      this.downloadData(data);
+    });
   }
 
   private downloadData(data: string): void {
     this.renderer.setAttribute(
       this.dlJsonAnchorElement,
       'href',
-      `data:text/json;charset=utf-8,${encodeURIComponent(data)}`
+      `data:text/plain;charset=utf-8,${encodeURIComponent(data)}`
```

Comment:
why this change? How would this change the original json download?

Reply:
Since we have yaml as well to download, I moved to plain text, I does not affect original download